bank_statement_parse/parserlib.py

```diff
@@ -0,0 +1,228 @@
+        """property setter"""
+        self['amount'] = transferred_amount
+
+    @property
+    def eref(self):
+        """property getter"""
+        return self['ref']
+
+    @eref.setter
+    def eref(self, eref):
+        """property setter"""
+        self['ref'] = eref
+        if not self.message:
+            self.name = eref
+
+    @property
+    def message(self):
+        """property getter"""
+        return self._message
+
+    @message.setter
+    def message(self, message):
+        """property setter"""
+        self._message = message
+        self.name = message
+
+    @property
+    def remote_owner(self):
+        """property getter"""
+        return self['partner_name']
+
+    @remote_owner.setter
+    def remote_owner(self, remote_owner):
+        """property setter"""
+        self['partner_name'] = remote_owner
+        if not (self.message or self.eref):
+            self.name = remote_owner
+
+    @property
+    def remote_account(self):
+        """property getter"""
+        return self['account_number']
+
+    @remote_account.setter
+    def remote_account(self, remote_account):
+        """property setter"""
+        self['account_number'] = remote_account
+
+    def __init__(self):
+        """Define and initialize attributes.
+
+        Not all attributes are already used in the actual import.
+        """
+        super(BankTransaction, self).__init__()
+        self.transfer_type = False  # Action type that initiated this message
+        self.execution_date = False  # The posted date of the action
+        self.value_date = False  # The value date of the action
+        self.remote_account = False  # The account of the other party
+        self.remote_currency = False  # The currency used by the other party
+        self.exchange_rate = 0.0
+        # The exchange rate used for conversion of local_currency and
+        # remote_currency
+        self.transferred_amount = 0.0  # actual amount transferred
+        self.name = ''
+        self._message = False  # message from the remote party
+        self.eref = False  # end to end reference for transactions
+        self.remote_owner = False  # name of the other party
+        self.remote_owner_address = []  # other parties address lines
+        self.remote_owner_city = False  # other parties city name
+        self.remote_owner_postalcode = False  # other parties zip code
+        self.remote_owner_country_code = False  # other parties country code
+        self.remote_bank_bic = False  # bic of other party's bank
+        self.provision_costs = False  # costs charged by bank for transaction
+        self.provision_costs_currency = False
+        self.provision_costs_description = False
+        self.error_message = False  # error message for interaction with user
+        self.storno_retry = False
+        # If True, make cancelled debit eligible for a next direct debit run
+        self.data = ''  # Raw data from which the transaction has been parsed
```

Comment:
Is this __init__ method still necessary now that everything is handled in properties mapping to dict entries?